oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -269,7 +274,17 @@ impl Runtime {
                     "{:?}: new ABI handle {} maps to {:?}",
                     node_id, candidate, half
                 );
+
+                let event_details = HandleCreated {
+                    node_id: node_id_to_primitive(node_id),
+                    handle: candidate,
+                    channel_id: half.get_id(),
+                };
+
                 node_info.abi_handles.insert(candidate, half);
```

Comment:
nit: I'd move this line of code before the line that creates `event_details`.

Reply:
I'm still new to Rust, but afaik that's not possible. This line moves the `half` variable out of scope, and I need to access it to get the `channel_id` for the event details. :)

Reply:
Yes. I thought maybe there is a way around it. It should be possible to send the  `introspection_event` first and then update the `node_info`. But it is not a big deal... :)

Reply:
Yes! I could do that. I guess my intention is to send the event _after_ the action has been performed, but it's my obsessive focus on details more than anything else. ^^